package-lock.json

```diff
@@ -17927,7 +17927,7 @@
         },
         "@types/yargs": {
           "version": "13.0.9",
-          "resolved": "https://registry.npmjs.org/@types/yargs/-/yargs-13.0.9.tgz",
+          "resolved": "https://npm.morph.int.tools.bbc.co.uk/@types/yargs/-/yargs-13.0.9.tgz",
```

Comment:
Should this be handled outside of this PR? Seems like this is on latest:
![image](https://user-images.githubusercontent.com/53564281/90385915-b4f86b80-e07b-11ea-8a7c-394f72fa94c0.png)

Reply:
Yeah - looks like somebody else did the same and it wasn't picked up in code review. Is this happening because of configuration on people's machines based on setup for other projects?